login.js

```diff
@@ -0,0 +1,72 @@
+// 이메일 input 요소 가져오기
+let emailInput = document.getElementById("email");
+let emailError = document.getElementById("email-error");
+
+// 비밀번호 input 요소 가져오기
+let passwordInput = document.getElementById("password");
+let passwordError = document.getElementById("password-error");
+
+// 비밀번호 토글 이미지 가져오기
+let passwordToggle = document.getElementById("password-toggle");
+
+let emailPattern = /^[a-zA-Z0-9._-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,}$/;
```

Comment:
상수로 작성하면 "정규식이구나"를 바로 알 수 있을 것 같아요.

```suggestion
const EMAIL_PATTERN = /^[a-zA-Z0-9._-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,}$/;
```